ui/package.json

```diff
@@ -25,6 +25,7 @@
         "dagre": "^0.8.5",
         "history": "^4.10.1",
         "js-yaml": "^4.1.0",
+        "linkify-it": "^5.0.0",
```

Comment:
We don't seem to use all the logic from this library: https://github.com/markdown-it/linkify-it. It also doesn't seem tree-shakeable given that it relies on prototype methods

I might suggest something smaller, in the spirit of #12059

Reply:
You're right about prototype but seems to me that the impact of this library is quite negligible compared to others we could use

---- 

### Linkify-it (current) 
![Linkify-it](https://github.com/user-attachments/assets/90270f11-4ddb-4393-9dcd-317c7bbb2d79)

### Linkifyjs (other I could find)
<img width="634" alt="Linkifyjs" src="https://github.com/user-attachments/assets/a1a75242-b9df-4f3b-9bf7-fe40c1fe3408">

---- 
Also a note about lazy and the project.. (I wrote it here first but let's discuss it [on the issue here](https://github.com/argoproj/argo-workflows/issues/12059#issuecomment-2310473252))

Reply:
I agree that the library is only partly tree shakable, but it looks quite small at the end. Should I keep looking for another lib ?